global_helpers/panther_oss_helpers.py

```diff
@@ -258,6 +259,26 @@ def aws_strip_role_session_id(user_identity_arn):
     return user_identity_arn
 
 
+def check_threshold(key: str,
+                    threshold: int = 10,
+                    expiry_seconds: int = 3600) -> bool:
+    hourly_error_count = increment_counter(key)
+    if hourly_error_count == 1:
+        set_key_expiration(key, time.time() + expiry_seconds)
+    # If it exceeds our threshold, reset and then return an alert
+    elif hourly_error_count >= threshold:
+        reset_counter(key)
+        return True
+    return False
+
+
+# Basic helpers
+
+
+def pattern_match(string_to_match: str, pattern: str):
+    return fnmatch(string_to_match, pattern)
```

Comment:
Not sure if `from panther_oss_helpers import pattern_match` is any better than `from fnmatch import fnmatch`

I'd vote to remove this one